bears/coffee_script/CoffeeLintBear.py

```diff
@@ -4,6 +4,11 @@
 from coalib.bears.requirements.NpmRequirement import NpmRequirement
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 from coalib.results.Result import Result
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+
+
+def _coffee_bool(py_bool):
+    return 'true' if py_bool else 'false'
```

Comment:
I believe you are taking a boolean and returning the corresponding string. Right? Instead of calling function you may just use `str(py_bool)` inline. For example [here at L215](https://github.com/coala-analyzer/coala-bears/pull/555/commits/7c2d8c52462b6e1fc37337335d198fd7d507cd7d#diff-a321f9a8f5f440c140740f55883d07e7R215)
You avoid save extra chars and also it's slightly more efficient. You're avoiding function call overheads. :) @Asnelchristian